### PR TITLE
OSDOCS-8970 RN for must-gather storage limit

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.15

Issue: [OSDOCS-8970](https://issues.redhat.com/browse/OSDOCS-8970)

Link to docs preview:[ Preview in Notable Technical Changes](https://71712--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-notable-technical-changes)

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by cbippley** [comment]
@kasturinarra Could you PTAL? Thank you!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -805,6 +805,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+A default storage limit of 30% has been added for data collected by the `oc adm must-gather` command. To adjust the default storage limit value a new flag has been added to the OpenShift CLI.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-15-release-notes.adoc:812]: * Probably should clarify what it is 30% of
* Better to use active voice, and write it in the perspective of the reader taking action. Added a suggestion but you can play with it however you'd like

```suggestion
A default limit of 30% of the storage capacity of the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```

**Review by bergerhoffer — COMMENTED** [review]
Quick tip, but I'd still send for normal peer review in case someone else has some other suggestions

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:812]: Thank you!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -805,6 +805,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+ A default limit of 30% of the storage capacity of the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```
Comment (ardaguclu) [inline release_notes/ocp-4-15-release-notes.adoc:812]: Would it make sense to mention that this limit is 30% percentage of the node's storage that container is running on. 

Also would it make sense to mention what will happen when the limit is exceeded or is it described in somewhere else such as in the link below?

Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:812]: Are you saying that the storage limit is not 30% of the container's storage, but 30% of the node's storage? 

What happens if the limit is exceeded is explained in the linked section, I don't think we need to add it to the release note.

Comment (ardaguclu) [reply release_notes/ocp-4-15-release-notes.adoc:812]: > Are you saying that the storage limit is not 30% of the container's storage, but 30% of the node's storage?

yes, it is node's storage

Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc]: @ardaguclu How does it look now?

**Comment by ardaguclu** [comment]
LGTM

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -814,6 +814,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
 
 For more information, see xref:../installing/cluster-capabilities.adoc#deployment-config-capability_cluster-capabilities[DeploymentConfig capability].
 
+[discrete]
+[id="ocp-4-15-must-gather-storage-default"]
+=== Must-gather storage limit default
+
+ A default limit of 30% of the storage capacity of the node for the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```
Comment (bburt-rh) [inline release_notes/ocp-4-15-release-notes.adoc:821]:
```suggestion
A default limit of 30% of the storage capacity of the node for the container has been added for data collected by the `oc adm must-gather` command. If necessary, you can use the `--volume-percentage` flag to adjust the default storage limit.
```
- I think the blank space at the beginning of the line is causing the entire paragraph to render as a code block in the preview:

image

File: release_notes/ocp-4-15-release-notes.adoc [hunk 4]
```diff
@@ -814,6 +814,14 @@ If you disable the `DeploymentConfig` cluster capability, the `deployer` service
...
+
+For more information, see xref:../support/gathering-cluster-data.adoc#must-gather-storage-limit_gathering-cluster-data[Changing the must-gather storage limit].
```
Comment (bburt-rh) [inline release_notes/ocp-4-15-release-notes.adoc:823]: - This xref doesn't seem to be opening to the correct location for me in the rendered preview.

**Review by bburt-rh — APPROVED** [review]
A couple of rendering issues that need to be fixed, but other than that, lgtm. Let me know if you want me to take another look.

File: release_notes/ocp-4-15-release-notes.adoc (hunk 4 above)
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc:823]: The section it links to only exists in the 4.15 branch so I _think_ that means it won't show up properly in the preview.

Comment (bburt-rh) [reply release_notes/ocp-4-15-release-notes.adoc:850]: Might be something in my Firefox browser settings. It appears to be working OK in Chrome. Setting this to resolved.

File: release_notes/ocp-4-15-release-notes.adoc (hunk 3 above)
Comment (cbippley) [reply release_notes/ocp-4-15-release-notes.adoc]: Fixed!

**Pull request merged**